Tests/Snapshotting+presentationLayer.swift

```diff
@@ -0,0 +1,41 @@
+// Created by Cal Stephens on 12/15/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import SnapshotTesting
+import UIKit
+import XCTest
+
+extension Snapshotting where Value == UIView, Format == UIImage {
+  /// Captures an image of the input `UIView`'s `layer.presentation()`,
+  /// which reflects the current state of any in-flight animations.
+  static func imageOfPresentationLayer(
+    precision: Float = 1,
+    traits: UITraitCollection = .init())
+    -> Snapshotting<UIView, UIImage>
+  {
+    SimplySnapshotting
+      .image(precision: precision, scale: traits.displayScale)
+      .pullback { (view: UIView) -> UIImage in
+        // Place the view in an on-screen window and then
+        // have Core Animation update the screen synchronously
+        let window = UIWindow()
+        window.isHidden = false
+        window.addSubview(view)
+
+        // Calling CATransaction.flush() makes Core Animation synchronously update
+        // the content being displayed on-screen, which gives our view a presentation layer.
+        CATransaction.flush()
+
+        // Now that the view is on-screen, it has a presentation layer:
+        guard let presentationLayer = view.layer.presentation() else {
+          fatalError("Presentation layer does not exist and cannot be snapshot")
+        }
```

Comment:
Is it possible to throw here so the test fails rather than crashing? Or does `pullback` not rethrow?

Reply:
`pullback` is not `rethrows` so we have to either `fatalError` or return an empty `UIImage`

an empty `UIImage` would probably be better here, since it would show up as a diff failure for the specific problematic test case and not block everything else from continuing